src/core/CoolLog.h

```diff
@@ -41,6 +41,8 @@
     Serial.print(F(level));                                                    \
     Serial.print(F(" "));                                                      \
     Serial.println(F(m));                                                      \
```

Comment:
replace `println` by `print` here as well

Reply:
and you add space after the two line